lib/ZwaveClient.js

```diff
@@ -650,7 +650,7 @@ function updateValueMetadata (zwaveNode, zwaveValue, zwaveValueMeta) {
     readable: zwaveValueMeta.readable,
     writeable: zwaveValueMeta.writeable,
     description: zwaveValueMeta.description,
-    label: zwaveValueMeta.label,
+    label: zwaveValueMeta.label || zwaveValue.propertyName, // when label is missing, re use propertyName. Usefull for webinterface
```

Comment:
@AlCalzone could the label be undefined or is this caused by an error on zwavejs? If it's the second case I would remove this so we can detect undefined labels and fix them on zwavejs

Reply:
The Label can be undefined if we don't set it in zwave-js. Your code should be prepared for that possibility.
But as I wrote in slack, we should find those locations and eliminate them.

Reply:
Ok so in poor words should I keep this change or remove this so we can detect undefined labels?

Reply:
we could have a suffix/prefix Property and help us still know what is, but not let it undifined
something like
```
label: zwaveValueMeta.label || ("%s - property", zwaveValue.propertyName),
```

Reply:
This could be another valid solution

Reply:
i think more beautifull would be:
```property: %s```